src/board.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include "board.h"
+#include "gui.h"
+
+Board::Board() : Board(9, false) {}
+
+Board::Board(size_t size, bool cpu) {
+  isCpu = cpu;
+  p1Score = 0;
+  p2Score = 0;
+  boardSize = size;
```

Comment:
so we're doing camelCase eh?

Reply:
will make everything snake :)

Reply:
Oh interesting, Google C++ style guide, functions are CapCamelCase

Reply:
will do snake for functions so we're on the same page 👍

Reply:
down for google's style guidelines

Reply:
imma do snake instead, I ain't ready for the UpperCamel life haha